tests/utilities/test_app.py

```diff
@@ -38,6 +38,9 @@ def __init__(
             log_color_system="256",
         )
 
+        # Let's disable all features by default
+        self.features = frozenset()
```

Comment:
let's make sure that by default our AppTest class has all features disabled - which should make integration tests faster in local environments, as they won't try to connect to the DevTools console even if `TEXTUAL=devtools` is enabled